_studio/shared/umc/codec/h265_dec/src/umc_h265_va_packer.cpp

```diff
@@ -701,19 +704,19 @@ void PackerVA::PackQmatrix(const H265Slice *pSlice)
 void PackerVA::PackAU(const H265DecoderFrame *frame, TaskSupplier_H265 * supplier)
 {
     if (!frame || !supplier)
-            throw h265_exception(UMC_ERR_NULL_PTR);
+        throw h265_exception(UMC_ERR_NULL_PTR);
 
-    H265DecoderFrameInfo const* sliceInfo = frame->m_pSlicesInfo;
-    size_t const sliceCount = sliceInfo->GetSliceCount();
-    if (!sliceCount)
-        return;
+    PackPicParams(frame, supplier);
```

Comment:
is ```frame``` really the same as
```
H265DecoderFrameInfo const* sliceInfo = frame->m_pSlicesInfo;
H265Slice *pSlice = sliceInfo->GetSlice(0);
H265DecoderFrame *pCurrentFrame = pSlice->GetCurrentFrame();
```
?

Reply:
Any slice must be a part of only one frame, it is logic error otherwise.

Reply:
I think yes, because:
void H265DecoderFrame::AddSlice(H265Slice * pSlice)
{
...
     pSlice->m_pCurrentFrame = this;
...
}

H265DecoderFrame *GetCurrentFrame(void) const {return m_pCurrentFrame;}

Reply:
ok